### PR TITLE
Init a pipeline for creating Grafana Cloud objects inside existing stacks.

### PR DESCRIPTION
**Pull request by avnes**
Pipeline for ticket https://github.com/dfds/cloudplatform/issues/2575

### CODE REVIEW

File: environments/sandbox/.terraform.lock.hcl
Comment (samidbb) [inline environments/sandbox/.terraform.lock.hcl:1]: Did we start committing .lock.hcl files into Git?

File: environments/demo/.terraform.lock.hcl
Comment (samidbb) [inline environments/demo/.terraform.lock.hcl:1]: Did we start committing .lock.hcl files into Git?

File: dashboards.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  json_files = fileset(path.module, "dashboards/*.json")
+  json_data = [for f in local.json_files : {
+    filename = f
+    content  = (file(f))
+    }
+  ]
+}
+
+module "dashboards" {
+  source      = "git::https://github.com/dfds/terraform-grafana-cloud.git//grafana_dashboard?ref=0.2.0"
```
Comment (samidbb) [inline dashboards.tf:11]: If I am to use to reference to dashboards in the alerts that we are going to create then probably need to consider to put the "loop" logic outside the grafana_dashboard instead so we can reference the dashboards by id.

File: .github/CODEOWNERS
[the remark this answers is not in the document]
Comment (avnes) [reply .github/CODEOWNERS:1]: Thanks, yes I recreated the repo and forgot to add the permissions back

File: dashboards.tf (hunk 1 above)
Comment (avnes) [reply dashboards.tf]: Support for that ahs now been added and is examplified by an output of dashboard UIDs

**Comment by avnes** [comment]
@samidbb dashboard title is now also available in the output

**Pull request merged**